src/Margay.cpp

```diff
@@ -381,6 +384,9 @@ void Margay::I2Ctest()
 	}
 
 	if(I2C_Test) Serial.println("PASS");
+
+	// make sure I2C Bus is returned to initial state
+	closeFarmGateI2C(initialStateI2C);
```

Comment:
Can open or close gate, so:
```suggestion
	farmGateI2C(initialStateExternI2C);
```